playbook/app/pb_kits/playbook/pb_table/subcomponents/_table_row.tsx

```diff
@@ -83,14 +83,14 @@ const TableRow = (props: TableRowPropTypes): React.ReactElement => {
       target instanceof SVGElement &&
       (target.matches("svg.pb_custom_icon") || target.closest("svg.pb_custom_icon"));
 
-        if (clickedCell || isIconClick) { 
+        if (clickedCell && isIconClick) {
```

Comment:
For this fix, did you test what would happen if they DON'T use an icon in that clickedCell? I'm assuming it will not work because it'll be looking for an icon..I think something like (clickedCell || (clickedCell && isIconClick)) might be a safer bet, but you'll have to try that